modules/visualisation/src/backend/index.ts

```diff
@@ -0,0 +1,47 @@
+import 'bluebird-global'
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+
+import { getOnBotMount } from '../module_lifecycle/on_bot_mount'
```

Comment:
If the module doesn't have complex lifecycle methods, you can put the code directly here in the index file